packages/giselle-engine/src/core/generations/generate-text.ts

```diff
@@ -469,6 +484,41 @@ export async function generateText(args: {
 				generation: completedGeneration,
 				onConsumeAgentTime: args.context.onConsumeAgentTime,
 			});
+
+			// necessary to send telemetry but not explicitly used
+			const {
+				langfuse: _langfuse,
+				trace: _trace,
+				span: _span,
+				generation: _generation,
+			} = createLangfuseTracer({
```

Comment:
Nit: If we need to keep these objects for telemetry, maybe we could write the code like this instead:

```suggestion
			const _langfuseTracer = createLangfuseTracer({
```

Reply:
Thank you, I confirmed that the telemetries arrived after applying the change suggestion above! ✅